po/fr.po

```diff
@@ -19,48 +19,48 @@ msgstr ""
 
 #: src/MainView.vala:43 src/MainView.vala:160
 msgid "Copied to clipboard"
-msgstr ""
+msgstr "Copié dans le presse-papiers"
 
 #: src/MainView.vala:48
 msgid "Site:"
-msgstr ""
+msgstr "Site :"
 
 #: src/MainView.vala:54
 msgid "GitHub"
-msgstr ""
+msgstr "GitHub"
 
 #: src/MainView.vala:60
 msgid "Site is not case-sensitive. “GitHub” equals “github”."
-msgstr ""
+msgstr "Le site n'est pas sensible à la casse. « GitHub » est équivalent à « github »."
 
 #: src/MainView.vala:63
 msgid "Cipher key:"
-msgstr ""
+msgstr "Clé de chiffrement :"
 
 #: src/MainView.vala:73
 msgid "correct horse battery staple"
-msgstr ""
+msgstr "Phrase secrète"
```

Comment:
In the Spanish translation, I translated those four words literally, since placeholders are usually meant for examples rather than descriptions. Do you think that would make sense? Or do most French websites put a description as a placeholder?

Reply:
@dar5hak I was not sure about the literal translation so I put a generic field description instead. Is this only a suit of words without any link between them?

Reply:
It's a actually a reference to https://xkcd.com/936/, but even for the many people who haven't necessarily seen the comic, the words signal that you can use any passphrase you can remember without it needing to be a complicated mess.

If you think that would work, you can still add a commit.